webcompat/views.py

```diff
@@ -263,6 +263,8 @@ def create_issue():
 
         if ab_active('exp') == 'form-v2':
             bug_form = get_form(form_data, form=FormWizard)
+            # TODO: remove this when the experiment has ended
+            form_data['extra_labels'].append('form-v2-experiment')
```

Comment:
I guess in one way or the other this full code will disappear anyway be in the case of rejecting or adopting the new form.